src/common/components/button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent, ReactElement } from 'react';
+import joinClassNames from 'classnames';
+
+import styles from './button.module.scss';
+
+const Button: FunctionComponent<{
+  className?: string;
+  children: string | ReactElement;
+}> = ({ children, className, ...rest }) => {
+  return (
+    <div className={joinClassNames(styles.buttonWrapper, className)}>
```

Comment:
Why don't you apply these class names directly on the `button` element?

Reply:
I think it's better to have wrapper, rather than modifying straight on the html5 `button`.
Sometime they need some padding / margin / customization / outside label